Providers/Modules/Plugins/PatchManagement/conf/patch_management.conf

```diff
@@ -4,7 +4,7 @@
   command /opt/microsoft/omsconfig/Scripts/PerformInventory.py --InMOF /etc/opt/microsoft/omsagent/conf/omsagent.d/patch_management_inventory.mof --OutXML /etc/opt/omi/conf/omsconfig/configuration/CompletePackageInventory.xml > /dev/null && cat /etc/opt/omi/conf/omsconfig/configuration/CompletePackageInventory.xml
   format tsv
   keys xml
-  run_interval 3h
+  run_interval 1h
```

Comment:
You need to also change [Makefile](https://github.com/Microsoft/PowerShell-DSC-for-Linux/blob/master/Makefile#L471) to update the module version.